src/sass/gi/partials/_gi-modal.scss

```diff
@@ -1,12 +1,21 @@
 .gi-app {
-  
+
   .va-modal {
     background-color: rgba($color-gray-dark, .8);
     padding: 10px;
 
+    @include media-maxwidth($small-screen) {
+      overflow-y: scroll;
+    }
```

Comment:
@U-DON I only enabled this for GIBCT as to not affect behavior in other apps, however I don't foresee any problems with applying this patch to all modals across vets-website

@jbalboni @tiffanybbrown-va thoughts on this PR? This PR will fix background-scrolling issues on all modals across vets-website